libs/components/ag-grid/src/lib/modules/ag-grid/cell-editors/cell-editor-lookup/cell-editor-lookup.component.ts

```diff
@@ -29,7 +28,7 @@ import { SkyAgGridLookupProperties } from '../../types/lookup-properties';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class SkyAgGridCellEditorLookupComponent
-  implements ICellEditorAngularComp, IPopupComponent<unknown>
+  implements ICellEditorAngularComp
```

Comment:
`IPopupComponent` is redundant b/c `ICellEditorAngularComp` implements it, and the deep import broke with `31.2`.